internal/cli/opsmanager/backup/config/describe.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+var describeTemplate = `PROJECT ID	CLUSTER ID	STATUS	STORAGE ENGINE
+{{.GroupID}}	{{.ClusterID}}	{{.StatusName}}	{{.StorageEngineName}}
+`
+
+type DescribeOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	store     store.BackupConfigGetter
+	clusterID string
+}
+
+func (opts *DescribeOpts) initStore() error {
+	var err error
+	opts.store, err = store.New(config.Default())
+	return err
+}
+
+func (opts *DescribeOpts) Run() error {
+	r, err := opts.store.GetBackupConfig(opts.ConfigProjectID(), opts.clusterID)
+	if err != nil {
+		return err
+	}
+
+	return opts.Print(r)
+}
+
+// mongocli ops-manager backup config describe clusterID [--projectId projectId]
+func DescribeBuilder() *cobra.Command {
+	opts := &DescribeOpts{}
+	cmd := &cobra.Command{
+		Use:     "describe [clusterID]",
```

Comment:
[nit] brackets ([]) for optional, for mandatory the standard is less than, greater than (<>)
```suggestion
		Use:     "describe <clusterID>",
```